docs/install/assets/eventing.md

```diff
@@ -0,0 +1,11 @@
+# Knative Serving installation files (0.21 releases)
+
+| File name | Description | Dependencies|
+| --- | --- | --- |
+| eventing-core.yaml | Knative Serving core components. |  none |
+| eventing-crds.yaml | Knative Serving core CRDs. | none |
```

Comment:
I would put crds.yaml above core.yaml, since crds.yaml needs to be install first, core depends on it

Reply:
+1

Reply:
I think the list is supposed to follow the order in the eventing release
<img width="287" alt="Screen Shot 2021-03-04 at 17 40 07" src="https://user-images.githubusercontent.com/33425661/110005691-aff5c380-7d10-11eb-8818-d2f71913e0ad.png">

Also it follows the same order as the equivalent table for serving: https://github.com/yanweiguo/docs/blob/c3b1e89048674b7cc235f8e59c715a41986aab5c/docs/install/assets/serving.md

If you feel it is necessary to change the order, it would be best to do it in both tables (they will be on the same page).